src/Symfony/Component/Stopwatch/StopwatchPeriod.php

```diff
@@ -25,44 +25,59 @@ class StopwatchPeriod
     /**
      * Constructor.
      *
-     * @param int $start The relative time of the start of the period (in milliseconds)
-     * @param int $end   The relative time of the end of the period (in milliseconds)
+     * @param int|float $start         The relative time of the start of the period (in milliseconds)
+     * @param int|float $end           The relative time of the end of the period (in milliseconds)
+     * @param bool      $morePrecision If true, time is stored as float to keep the original microsecond precision
      */
     public function __construct($start, $end)
     {
-        $this->start = (int) $start;
-        $this->end = (int) $end;
+        $this->start = $start;
```

Comment:
yep. and currently, passing eg strings will cause odd working of method, while previously they would be converted in constructor already